offchain/README.md

```diff
@@ -0,0 +1,9 @@
+# Cartesi Node Reference Implementation
+
+This file is a work in progress. It will soon document how to build, test and
+overall handle a Cartesi Node.
```

Comment:
```suggestion
This file is a work in progress. It will soon document how to build, test and overall handle a Cartesi Node.
```

We shouldn't break lines in the middle of the sentences. Instead, we should break after every dot. This makes diffs easier to read.